server/index.js

```diff
@@ -1,13 +1,29 @@
 /* eslint-disable no-console */
 const express = require('express');
 const path = require('path');
+const db = require('../database');
 
 const app = express();
 const PORT = process.env.PORT || 3000;
 
 const PUBLIC_DIR = path.resolve(__dirname, '..', 'client', 'public');
 
-app.use(express.static(PUBLIC_DIR));
+app.use('/', express.static(PUBLIC_DIR));
+
+app.use('/api/:productId', (req, res) => {
```

Comment:
Perhaps consider an api endpoint change (i.e. /api/images/:productId) down the track (though perhaps the proxy can take care of that)? 🤷‍♂️

Reply:
I like the idea, I'll make a ticket for when I move back to the API interactions.